pkg/kubelet/hyper/hyperclient.go

```diff
@@ -396,19 +396,14 @@ func (client *HyperClient) ListContainers() ([]HyperContainer, error) {
 	return result, nil
 }
 
-func (client *HyperClient) Info() (map[string]interface{}, error) {
```

Comment:
@YaoZengzeng you should return something like HyperInfo, you can define it in hyper/types.go